lib/vm/src/trap/traphandlers.rs

```diff
@@ -341,7 +341,7 @@ cfg_if::cfg_if! {
                     None => return EXCEPTION_CONTINUE_SEARCH,
                 };
                 let jmp_buf = info.handle_trap(
-                    (*(*exception_info).ContextRecord).Rip as *const u8,
+                    (*(*exception_info).ContextRecord).Eip as *const u8,
```

Comment:
I believe Rip exists in 64-bit Windows environments. Can we have a cfg flag to choose one or the other (`Rip` in 64 bit Windows, `Eip` in 32 bit ones) depending on the bits of the system?